target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -515,9 +515,13 @@ ramips_setup_macs()
 	netgear,wnce2001|\
 	ohyeah,oy-0001|\
 	phicomm,k2g|\
-	skylab,skw92a)
+	skylab,skw92a|\
+	totolink,lr1200)
 		wan_mac=$(mtd_get_mac_binary factory 0x2e)
 		;;
+	adslr,g7)
+		wan_mac=$(mtd_get_mac_binary factory 0x4)
+		;;
```

Comment:
Considering that this router has a label_mac stored at 0xe00c, it's unlikely that vendor decided to use 0xe00c for lan_mac while also using mac for wifi card as wan_mac.
I'd prefer to keep this router to use lan_mac+1 as wan_mac and will patch this away when applying.

Reply:
Okay, I accept that reasoning.

Reply:
BTW: If you want to get rid of the lengthy explanation, it's okay if you just use my Signed-off-by for this change.